### PR TITLE
DOCS-1270: Improve generic

### PR DESCRIPTION
**Pull request by JessamyT**
Removed third bullet point since I believe that case falls under Advanced MR

### CODE REVIEW

File: docs/components/generic/_index.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ no_list: true
 
 The _generic_ component {{< glossary_tooltip term_id="subtype" text="subtype" >}} is for custom components that are incompatible with any of the other component APIs.
 
+For example, if you have an LED display, it does not make sense to implement it as an arm, camera, or any other existing component subtype because an LED display does not have joints or capture images.
+
 There are no built-in generic component models (other than `fake`).
 Use generic for a [modular resource](/modular-resources/) model that represents a unique type of hardware.
```
Comment (andf-viam) [inline docs/components/generic/_index.md:20]: I feel like `unique` is the word that most helps me here, what if we bumped it to the leading sentence with something like this:

```suggestion
The _generic_ component {{< glossary_tooltip term_id="subtype" text="subtype" >}} enables you to add support for unique types of hardware that do not already have an [appropriate API](/program/apis/#component-apis) defined for them.

For example, when using an [arm component](/components/arm), it makes sense to use the [arm API](/components/arm/#api), which provides specific functionality an arm component needs, such as moving to position or stopping movement.
But if you want to use an LED display, you need very different functionality that isn't currently exposed in any API.

Instead, you can use the generic component API to add support for your unique type of hardware, like LED displays, to your smart machine.
```

Also, is this the right place for this sentence?
```
There are no built-in generic component models (other than `fake`).
```

After convincing the user to use generic, this sentence doesn't seem directly related as is.

File: docs/modular-resources/_index.md [hunk 2]
```diff
@@ -59,18 +59,20 @@ If none of the existing modular resources in the Viam registry support your use
...
 To create a new modular resource:
 
-1. [Code a new resource model](/modular-resources/create/) by implementing all methods for the component's or service's [standardized APIs](/program/apis/).
+1. [Code a new resource model](/modular-resources/create/) by implementing all methods for the component's or service's [standardized API](/program/apis/).
 1. Package your modular resource or modular resources as a {{< glossary_tooltip term_id="module" text="module" >}} and [upload the module to the Viam registry](/modular-resources/upload/) to make it available for deployment to robots.
    You can upload _private_ modules for your [organization](/manage/fleet/organizations/) or _public_ modules.
 1. Once you have uploaded your module to the registry, you can [deploy and configure the module](/modular-resources/configure/) from [the Viam app](https://app.viam.com/).
    You can test your added resource using the [**Control** tab](/manage/fleet/#remote-control) and [program](/program/) it with Viam's Go or Python SDKs.
 
+### Unique cases
```
Comment (andf-viam) [inline docs/modular-resources/_index.md:72]: This header works for me, thanks!

File: docs/modular-resources/_index.md [hunk 3]
```diff
@@ -59,18 +59,20 @@ If none of the existing modular resources in the Viam registry support your use
...
+
+If your hardware won't work with any of the standard component APIs, you can use the [generic API](/components/generic/).
```
Comment (andf-viam) [inline docs/modular-resources/_index.md:74]: Similar to Naomi's comment, I'd like to move away from "won't work" (= Viam doesn't work). What about:

```suggestion
If you are using unique hardware that does not already have an [appropriate API](/program/apis/#component-apis) defined to support it, you can use the [generic API](/components/generic/) to add support for that unique hardware type to your smart machine.
```

**Review by andf-viam — APPROVED** [review]
LGTM with some language suggestions. Thank you!

File: docs/components/generic/_index.md [hunk 4]
```diff
@@ -12,7 +12,12 @@ no_list: true
 # SMEs:
 ---
 
-The _generic_ component {{< glossary_tooltip term_id="subtype" text="subtype" >}} is for custom components that are incompatible with any of the other component APIs.
+The _generic_ component {{< glossary_tooltip term_id="subtype" text="subtype" >}} enables you to add support for unique types of hardware that do not already have an [appropriate API](/program/apis/#component-apis) defined for them.
+
+For example, when using an [arm component](/components/arm), it makes sense to use the [arm API](/components/arm/#api), which provides specific functionality an arm component needs, such as moving to position or stopping movement.
+But if you want to use an LED display, you need very different functionality that isn't currently exposed in any API.
+
```
Comment (npentrel) [inline docs/components/generic/_index.md:19]:
```suggestion
```

File: docs/components/generic/_index.md [hunk 5]
```diff
@@ -12,7 +12,11 @@ no_list: true
 # SMEs:
 ---
 
-The _generic_ component {{< glossary_tooltip term_id="subtype" text="subtype" >}} is for custom components that are incompatible with any of the other component APIs.
+The _generic_ component {{< glossary_tooltip term_id="subtype" text="subtype" >}} enables you to add support for unique types of hardware that do not already have an [appropriate API](/program/apis/#component-apis) defined for them.
+
+For example, when using an [arm component](/components/arm), it makes sense to use the [arm API](/components/arm/#api), which provides specific functionality an arm component needs, such as moving to position or stopping movement.
```
Comment (npentrel) [inline docs/components/generic/_index.md:17]:
```suggestion
For example, when using an [arm component](/components/arm)/, it makes sense to use the [arm API](/components/arm/#api), which provides specific functionality an arm component needs, such as moving to position or stopping movement.
```

File: docs/components/generic/_index.md [hunk 6]
```diff
@@ -12,7 +12,11 @@ no_list: true
 # SMEs:
 ---
 
-The _generic_ component {{< glossary_tooltip term_id="subtype" text="subtype" >}} is for custom components that are incompatible with any of the other component APIs.
+The _generic_ component {{< glossary_tooltip term_id="subtype" text="subtype" >}} enables you to add support for unique types of hardware that do not already have an [appropriate API](/program/apis/#component-apis) defined for them.
+
+For example, when using an [arm component](/components/arm)/, it makes sense to use the [arm API](/components/arm/#api), which provides specific functionality an arm component needs, such as moving to position or stopping movement.
```
Comment (npentrel) [inline docs/components/generic/_index.md:17]:
```suggestion
For example, when using an [arm component](/components/arm/), it makes sense to use the [arm API](/components/arm/#api), which provides specific functionality an arm component needs, such as moving to position or stopping movement.
```

**Pull request merged**